### DISCUSSION TITLE
publish website using quarto

### DISCUSSION THREAD

**Discussion by Yalemzewod** [quarto-dev/quarto-cli#4976] [Q&A]
hi folks,
I am requesting a good Quarto guide to develop a personal website (portfolio) using quarto.

Thanks,
Yalem

**Comment by rgaiacs** [comment]
Hi @yalemz, what is https://quarto.org/docs/websites/ missing?

**Reply by Yalemzewod** [reply]
Hi @rgaiacs,
Thank you for your prompt response. I am following this and [samanthacsik](https://ucsb-meds.github.io/creating-quarto-websites/) guide. However, my directory is not tracked by git.

**Reply by rgaiacs** [reply]
Do you have Git installed? Are you using RStudio? Did you perform step 5 in https://ucsb-meds.github.io/creating-quarto-websites/ ?

```bash
git init
```

**Reply by Yalemzewod** [reply]
error: failed to push some refs to 'https://github.com/yalemz/yalemzassefag.github.io.git'

**Reply by rgaiacs** [reply]
Can't help you without a full log of the error. Looks like to me that https://github.com/yalemz/yalemzassefag.github.io.git has a initial commit that is not present in you local Git repository. My suggestions

1. Delete https://github.com/yalemz/yalemzassefag.github.io.git
2. Create a new repository at GitHub called `yalemz.github.io` **without any commit**. This name is required for GitHub Pages to publish using https://github.com/yalemz/yalemz.github.io.
3. Update your remote branch reference on your local machine.
